vault/README.md

```diff
@@ -0,0 +1,77 @@
+# Storage Inventory storage management service (vault)
+
+## configuration
+See the [cadc-tomcat](https://github.com/opencadc/docker-base/tree/master/cadc-tomcat) image docs
+for expected deployment and general config requirements. The `vault` war file can be renamed
+at deployment time in order to support an alternate service name, including introducing 
+additional path elements (see war-rename.conf).
+
+Runtime configuration must be made available via the `/config` directory.
+
+### catalina.properties (cadc-tomcat)
```

Comment:
remove `(cadc-tomcat)` from this heading

remove documented SQLGenerator since it is not needed.

for connection pool, I would name props `org.opencadc.vault.nodes.{prop}` since a vospace will also need a `uws` pool and vospace is too vague. 

there is an unclosed triple-tickmark box below that is messing with formatting